proto/executor.proto

```diff
@@ -142,6 +142,8 @@ message Join {
 	repeated Expr right_conditions = 10;
 	repeated Expr other_conditions = 11;
 	repeated Expr other_eq_conditions_from_in = 12;
+	repeated Expr left_null_aware_join_keys = 13;
+	repeated Expr right_null_aware_join_keys = 14;
```

Comment:
It seems these fields are equal to `left_join_keys` and `right_join_keys`. How about adding a boolean value to mark whether it is a null aware join?